### ISSUE TITLE
Other classes?

### ISSUE THREAD

**Issue by tiberium24** [BoPang1996/TubeTK#2]
Hello,

Does this algorithm work for other classes such as cars?

**Comment by Ukhupacha** [comment]
I don't think so, the datasets on the paper are MOT17 and JTA. Both are pedestrian datasets, have you been able to reproduce the results?

**Comment by tiberium24** [comment]
I have not tried to reproduce but I have a different application so I was wondering if I had to retrain for another dataset with cars. I believe that's the case, right?

**Comment by Ukhupacha** [comment]
You would need to modify the code, giving that you have to add another class. At the moment is it only(background, and person). I've been trying to reproduce the code without success.

**Comment by BoPang1996** [comment]
Right now, the provided model (weight) is trained on pedestrian data. So if you want to track cars, you need to retrain it. 

The good news is that we are training it on cars. And we will share the model after we finish it.

**Comment by tiberium24** [comment]
May I also ask you how you handled some of the detections in MOT dataset in terms of confidence?
My problem is that some the det.txt file has strange confidence values ranges (some are negative, some are above 1)
example:
3,-1,1021.2,436.37,31.49,96.47,**-0.18888**,-1,-1,-1

Why is it -0.188?

**Comment by BoPang1996** [comment]
TubeTK does not use detection results.
